core/devdoc/outprocess_module_requirements.md

```diff
@@ -0,0 +1,204 @@
+Binding for Out of Process Azure IoT Gateway Modules
+====================================================
```

Comment:
On the diagram of the on-out-process-gateway-modules.md didn't we call this Proxy? Why don't we call this Proxy here on the document name and tittle?